content/en/users/online-storage/_index.md

```diff
@@ -4,34 +4,37 @@ linkTitle: "Online Storage"
 type: docs
 weight: 55
 description: >
-  Store, share and access your files and their metadata on a global scale
+  EGI Online Storage services
 ---
 
-The EGI Online Storage includes a wide category of services which allows
-storing, sharing and accessing files on the EGI infrastructure. The service
-comprises different categories of storage services depending on the technology
-and usage that is foreseen. The 3 following service offerings are available:
+## Overview
 
-- [Grid Storage](grid-storage/)
-- [Object Storage](object-storage/)
-- [Block Storage](../cloud-compute/storage/#block-storage), which is described
-  under the Cloud Compute section
+EGI Online Storage includes a wide category of services that allow
+users to **store, share and access data in the cloud**, using the EGI Cloud
```

Comment:
i would not use the phrase "store, share and access data in the cloud," as the Grid Storage is mostly for HTC

Reply:
the same for "using the EGI Cloud infrastructure" better to remove Cloud

Reply:
But grid storage is still accessed over the internet, right? The storage is offered by (mostly OpenStack) IaaS providers, right? So it is in the cloud.

Reply:
The providers of grid storage + HTC could be also part of the Fedcloud and offer OpenStack as well, but it does not mean that the storage is provisioned via VMs ( most of the time is via bare metal nodes).

Reply:
I understand. But I still think of that as a cloud service. It is the same with commercial cloud service providers. Everything that is not on-prem is a cloud service. The expression "allow users to store, share and access data in the cloud" is generic, and covers all storage solutions EGI offers, then for each of the services offered you have a more specific summary under "What is it?" of the storage service's page. . But if you still think this is inaccurate, I can just drop the term "cloud".

Reply:
I think that in our community grid storage is definitely expected to be different from the cloud stuff. I would also be in favour of being more generic here.

Reply:
So more generic means without the term "cloud"?

Reply:
Fixed